js/app/downloader.js

```diff
@@ -10,6 +10,8 @@ const Soup = imports.gi.Soup;
 const Config = imports.app.config;
 const Utils = imports.app.utils;
 
+const CHUNK_SIZE = 1024 * 128;
```

Comment:
I don't want to bikeshed on the number, but I'd probably either downsize this to 64 or up it to 256 kB, with a preference for the latter.